app/src/components/map/Timebar.jsx

```diff
@@ -113,14 +127,22 @@ class Timeline extends Component {
       .attr('class', css['c-timeline-inner-brush'])
       .call(this.innerBrushFunc);
 
-    // mmove both brushes to initial position
-    this.outerBrushFunc.move(this.outerBrush, [0, width]);
-    this.redrawInnerBrush(this.props.filters.timelineInnerExtent);
-
     // no need to keep brush overlays (the invisible interactive zone outside of the brush)
     this.outerBrush.select('.overlay').remove();
     this.outerBrush.select('.selection').attr('cursor', 'default');
+    this.outerBrush.selectAll('.selection').classed(css['c-timeline-outer-brush-selection'], true);
     this.innerBrush.select('.overlay').remove();
+    this.innerBrush.select('.selection').classed(css['c-timeline-inner-brush-selection'], true);
+    innerBrushLeftCircle = this.innerBrush.append('circle');
+    innerBrushRightCircle = this.innerBrush.append('circle');
+    this.innerBrush.selectAll('circle')
+      .attr('cy', height / 2)
+      .attr('r', 5) // this would be better in the css but Safari is stupid
```

Comment:
I'm not sure your link is talking about CSS. It's saying that some attributes can be used for styling but not all of them can be set with CSS. If you check the [MDN's reference](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference) of CSS rules, you won't find `r`.

Anyway, I just wanted you to remove the comment 😉.

Regarding, testing in other browsers, that's a task every one should do while developing. Unfortunately, some deadlines prevent us from doing a whole testing. Count on me to tell you if it's broken on Safari.